CMakeLists.txt

```diff
@@ -22,15 +24,33 @@ set(tinyobjloader-examples-objsticher
 	${TINYOBJLOADEREXAMPLES_DIR}/obj_sticher/obj_sticher.cc
 	)
 
+#Install destinations
+set(TINYOBJLOADER_CMAKE_DIR lib/cmake)
```

Comment:
Distributions certainly package header-only libraries if users want them... Eigen is the best known example.